docs/api_reference/components/entity.md

```diff
@@ -1,6 +1,3 @@
-Entity
-======
+# Entity
```

Comment:
Do we need `automodule` here?

Reply:
Missed that file to update. It is now updated.